metrics.go

```diff
@@ -7,26 +7,29 @@ import (
 	"io"
 	"os"
 	"strconv"
+	"strings"
 	"syscall"
 
 	"github.com/pkg/errors"
 )
 
-type logLine struct {
-	Status    string `json:"status"`
-	Bytes     string `json:"bytes"`
-	BytesSent string `json:"bytes_sent"`
-	Hostname  string `json:"hostname"`
-}
-
 var (
 	filepath string
 )
 
-func (l logLine) getBytes() int {
-	bytes, _ := strconv.Atoi(l.Bytes)
+func sanitizeValue(label string, value string) string {
+	switch label {
```

Comment:
case for `hostname` to strip any port number suffix which has been seen in the wild.